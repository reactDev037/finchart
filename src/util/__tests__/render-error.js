import renderError from '../render-error';

describe('renderError', () => {
  it('renders', () => {
    const wrapper = renderError;
    expect(wrapper).toMatchSnapshot();
  });
});
