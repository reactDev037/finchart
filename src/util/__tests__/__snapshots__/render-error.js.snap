// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderError renders 1`] = `Object {}`;
